pkg/controller/bootstrap/bootstrap.go

```diff
@@ -234,3 +251,14 @@ func parseManifests(filename string, r io.Reader) ([]manifest, error) {
 		manifests = append(manifests, m)
 	}
 }
+
+func podmanCreateUseProxy(imageURL string) error {
+	command := "podman"
+	args := []string{"create", "--http-proxy", imageURL}
```

Comment:
should be a podman pull...

add the environment variables from above to the cmd context here...

cmd.Env = append("HTTP_PROXY=something", "HTTPS_PROXY=something_else")